Libraries/LibGUI/AutocompleteProvider.cpp

```diff
@@ -64,11 +62,8 @@ class AutoCompleteSuggestionModel final : public GUI::Model {
                 return suggestion.completion;
             }
             if (index.column() == Column::Icon) {
-                // TODO: Have separate icons for fields, functions, methods etc
-                // FIXME: Probably should have different icons for the different kinds, rather than for "c++".
-                if (suggestion.kind == CompletionKind::Identifier)
-                    return *s_cplusplus_icon;
-                return *s_cplusplus_icon;
+                // TODO
+                return {};
```

Comment:
I could either just return the C++ icon for `Language::Cpp`, or try to design a few (likely _bad_) icons, we currently really need two right now: `Cpp`+`Identifier`, `Unspecified`+`Identifier`

Reply:
Make the best icon you can and we'll iterate :)

Reply:
I'll try then.

Reply:
Icons are hard, evidently.
I hope someone that doesn't entirely suck at pixel art makes a good-looking set one day lol